File: rb/test/exists_test.rb
# EnergyChartsApi2 SDK exists test

require "minitest/autorun"
require_relative "../EnergyChartsApi2_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = EnergyChartsApi2SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- EnergyChartsApi2 SDK exists test

local sdk = require("energy-charts-api2_sdk")

describe("EnergyChartsApi2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
